ESP32-G.yaml

```
@@ -0,0 +1,48 @@
+substitutions:
+  name: esp32g-proxy
+
+esphome:
+  name: ${name}
+  name_add_mac_suffix: true
+  project:
+    name: esphome.bluetooth-proxy
+    version: "1.0"
+
+esp32:
+  board: esp32dev
+  framework:
+    type: arduino
+
+ethernet:
+  type: LAN8720
+  mdc_pin: GPIO23
+  mdio_pin: GPIO18
+  clk_mode: GPIO17_OUT
+  phy_addr: 1
+  power_pin: GPIO5
+
+api:
+logger:
+ota:
+
+dashboard_import:
+  package_import_url: github://esphome/bluetooth-proxies/esp32g.yaml@main
```

This filename does not match
```suggestion
  package_import_url: github://esphome/bluetooth-proxies/ESP32-G.yaml@main
```